### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by HelenHTech**
## What?

Update dependencies
🐿 v2.12.0

## Why? 

Required to fix failing heroku deployment build

### CODE REVIEW

File: .snyk [hunk 1]
```diff
@@ -0,0 +1,122 @@
+# Snyk (https://snyk.io) policy file, patches or ignores known vulnerabilities.
```
Comment (stephaniepieri) [inline .snyk:1]: Should this file be committed?

File: .gitignore [hunk 2]
```diff
@@ -4,3 +4,4 @@ dist
 *.dat
 *.txt
 node_modules
+node_modules
```
Comment (stephaniepieri) [inline .gitignore:7]: Looks like this has duplicated ^

File: .snyk (hunk 1 above)
Comment (HelenHTech) [reply .snyk:1]: Yep - It's fine to commit. When other engineers clone the branch locally, your code will require this file to see which vulnerabilities have no fixes available.

Comment (stephaniepieri) [reply .snyk:1]: Got it, thanks for explaining :-)

**Pull request merged**